src/client.js

```diff
@@ -67,13 +67,13 @@ function Client(manager, name, config) {
 
 	var client = this;
 
-	if (!client.config.token) {
-		client.updateToken(function() {
-			client.manager.updateUser(client.name, {token: client.config.token});
-		});
-	}
-
 	if (config) {
+		if (config.token) {
```

Comment:
Huh, yeah I think I made a mistake there. I need to switch it back to `!`. It is a problem with `config` being undefined indeed.